docs/.templates/integration/schema.json

```
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories_list": {
```

```suggestion
        "categories": {
```

I think it is ok without `_list`